### PR TITLE
Requisition department

### LINKED ISSUE #170: Quartermaster sprites

### LINKED ISSUE #172: Quartermaster ftl

### LINKED ISSUE #173: Quartermaster spawnpoint

### LINKED ISSUE #174: Cargo Tech sprites

### LINKED ISSUE #176: Cargo Tech ftl

### LINKED ISSUE #177: Cargo Tech spawnpoint

### LINKED ISSUE #178: Mess Tech sprites

### LINKED ISSUE #180: Mess Tech ftl

### LINKED ISSUE #181: Mess Tech spawnpoint

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR
Adds Requisition department, three new roles for it: Quartermaster, Cargo Technician and Mess Technician and clothes for them.

## Why / Balance
Resolves #170, resolves #172, resolves #173, resolves #174, resolves #176, resolves #177, resolves #178, resolves #180, resolves #181

## Media

image_2023-11-16_22-50-55

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Tunguso4ka
- added: Requisition department has been added to the game.

### CODE REVIEW

**Comment by Tunguso4ka** [comment]
Ready to review.

**Comment by Tunguso4ka** [comment]
wait. why the fuck wo uniform changed. oohh

**Comment by Tunguso4ka** [comment]
Ready to review.

**Comment by Tunguso4ka** [comment]
test problem dont really looks like mine problem.

File: Resources/Textures/_CM14/Objects/Clothing/Head/Hats/cargocap.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from cmss13 at commit https://github.com/cmss13-devs/cmss13/commit/05eaa3484b1a35aa759300bcfc8f0119f009daae",
```
Comment (DrSmugleaf) [inline Resources/Textures/_CM14/Objects/Clothing/Head/Hats/cargocap.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/05eaa3484b1a35aa759300bcfc8f0119f009daae/icons/obj/items/clothing/cm_hats.dmi and https://github.com/cmss13-devs/cmss13/blob/05eaa3484b1a35aa759300bcfc8f0119f009daae/icons/mob/humans/onmob/head_1.dmi",
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Comment by Tunguso4ka** [comment]
i really dont know why it throwing DebugAssertException on SpawnAndDeleteAllEntitiesInTheSameSpot.

**Comment by Tunguso4ka** [comment]
where my debug assert exception.

**Comment by DrSmugleaf** [comment]
> where my debug assert exception.

I reran tests to see if it was a random fail, looks good, thank you for the PR

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**